src/pages/students/[student_id]/index.tsx

```diff
@@ -11,12 +15,22 @@ const ViewStudentPage = () => {
     { enabled: Boolean(student_id) }
   );
 
+  const { mutate } = trpc.archiveStudent.useMutation();
+
+  const archiveStudent = async () => {
+    mutate({
+      student_id: student?.student_id || "",
```

Comment:
we should probably do a `if (!student) { return }` check above instead of defaulting to an empty string, since that would throw an error (TS will then realize that `student` must be defined in the code that follows)

Reply:
I modified this as you said.